algos/src/sample_rate_conversion.rst

```diff
@@ -7,65 +7,65 @@ Introduction
+to move the processing operations to the lowest sample rate possible and
+omit computing of intermediate results that would be discarded. The
+benefit of polyphase conversion is its capability to scale to very high
+quality like true 24-bit studio quality, since the filtering is a
 linear operation and the performance depends on the time-invariant
 filter characteristics. The algorithm does not limit the audio
 conversion quality.
 
-The SRC component is synchronous type that converts the rates with
+The SRC component is a synchronous type that converts the rates with
 exact rational M/N fraction and cannot adjust for any small drift of
-sample rate. Per every call to SRC the algorithm consumes exactly N
+the sample rate. Per every call to SRC, the algorithm consumes exactly N
 input samples and M output samples.
 
-As an example if input to SRC is 11025 Hz and output is 48000 Hz the
-fraction for conversion is 640/147. For every 147 input samples there
-are 640 output samples at 48 kHz. Such processing block would require
+As an example, if input to SRC is 11025 Hz and output is 48000 Hz, the
+fraction for conversion is 640/147. For every 147 input samples, there
+are 640 output samples at 48 kHz. Such a processing block would require
 13.3 ms of buffering. To shorten the latency and ease the conversion
-fractions some of the conversions are executed in two stages. The
+fractions, some of the conversions are executed in two stages. The
 fraction 640/174 can be factored as 32/21*20/7. With the two fractions
-approach the SRC will input with 21 frames granularity and output with
-20 frames granularity. The internal buffer between the stages places
-an internal constraing for processing block sizes. Still the approach
+approach, the SRC will input with 21 frames of granularity and output with
+20 frames of granularity. The internal buffer between the stages places
+an internal constraint for processing block sizes. Still, the approach
 provides much shorter latency than using a single fraction.
 
-In some cases it might be possible to design and use a converter that
-is intentionally non-exact. E.g. 48000/11000 conversion that is a lot
-easier faction 48/11 and provides much lower SRC latency. But use of
-such approximation with as low as 0.2% error would result to
-systematic slow drift of audio presentation so it is not
-recommended. Fortunately conversions in the 48 kHz family rates such
-as 32 kHz to 48 kHz is much lower latency with the 3/2 fraction with
-need for only 63 us additional buffer.
+In some cases, it might be possible to design and use a converter that
+is intentionally non-exact, such as a 48000/11000 conversion that has an
+easier faction of 48/11 and provides much lower SRC latency. But use of
+such approximation with as low as 0.2% error would result in a
+systematic slow drift of audio presentation so it is not recommended.
+Fortunately, conversions in the 48 kHz family rates such as 32 kHz to 48 kHz
```

Comment:
@singalsu Please review the last sentence in this paragraph.

Reply:
It's much better than my version but you need the preserve the microseconds in "63 us".